__tests__/api.test.js

```diff
@@ -169,6 +169,51 @@ describe("POST API - insert a comment row using review_id and author FK", () =>
     })
 })
 
+describe("DELETE API - delete comment by id", () => {
+    test("200: ID exists - delete comment by id", () => {
+        return request(app)
+            .delete("/api/comments/2")
+            .expect(200)
+            .then(({body}) => {
+                const expected = {
```

Comment:
When we are deleting, there is a status code `204` - this is a successful version of 404 in a way - the content cannot be found because it has been successfully deleted 

When expecting this code, there is no need to expect any further response, we can simply leave it at: 
`.expect(204);`